lib/tmuxinator/window.rb

```diff
@@ -34,13 +34,16 @@ def initialize(window_yaml, index, project)
 
     def build_panes(panes_yml)
       Array(panes_yml).map.with_index do |pane_yml, index|
-        if pane_yml.is_a?(Hash)
-          pane_yml.map do |_name, commands|
-            Tmuxinator::Pane.new(index, project, self, *commands)
-          end
-        else
-          Tmuxinator::Pane.new(index, project, self, pane_yml)
-        end
+        commands =  case pane_yml
+                    when Hash
+                      pane_yml.values.first
```

Comment:
Alright, now that I've read that issue and understand the premise a little bit more, I think your approach here with using `first` is right. I'm going to pull this down and play with it a little just to make sure, then I'll come back and give it a :+1: